missionbit/settings.py

```diff
@@ -52,7 +52,7 @@
     "crispy_forms",  # <- Crispy forms
     "coverage",  # <- for testing
     "salesforce",  # <- salesforce database
-    'django_q', # <- For queueing tasks
+    "django_q",  # <- For queueing tasks
```

Comment:
django_q is missing from requirements.txt

Reply:
added